composer.json

```diff
@@ -319,7 +319,8 @@
             },
             "drupal/inline_entity_form": {
                 "Fixes field label when a new entity is created inline (https://www.drupal.org/project/inline_entity_form/issues/2842744)":
-                "https://www.drupal.org/files/issues/inline_entity_form-no_label_required_field_with_no_entries-2842744.patch"
+                "https://www.drupal.org/files/issues/inline_entity_form-no_label_required_field_with_no_entries-2842744.patch",
```

Comment:
The patch specifics should be on one line. I know this problem predates this PR but lets fix it now please.